pyproject.toml

```diff
@@ -134,6 +105,33 @@ exclude = [
     "doc/_build",
 ]
 
+[tool.uv]
```

Comment:
this should wait on https://github.com/astral-sh/uv/issues/5632, i.e. we shouldn't need to collapse all the dev dependencies into a single list